src/pyfaf/actions/releaseadd.py

```diff
@@ -66,7 +67,8 @@ def run(self, cmdline, db):
         return 0
 
     def tweak_cmdline_parser(self, parser):
-        parser.add_opsys()
-        parser.add_opsys_release()
-        parser.add_argument("-s", "--status", default="ACTIVE",
-                            help="ACTIVE, UNDER_DEVELOPMENT or EOL")
+        parser.add_opsys(required=True, helpstr="operating system")
+        parser.add_argument("--opsys-release", required=True,
```

Comment:
Why this instead of the call to `add_opsys_release()`?

Reply:
Because `add_opsys_release()` adds a SelectField with _existing_ releases to the web UI form. We're adding a whole new release here so we need a plain TextField to get its name/number/version.

Reply:
Ah, ok, makes sense.